linkerd/app/integration/Cargo.toml

```diff
@@ -44,7 +44,7 @@ tower = "0.1"
 tower-grpc = { version = "0.1", default-features = false, features = ["protobuf"] }
 tracing = "0.1.9"
 tracing-futures = "0.1"
-webpki = "0.21"
+webpki = "=0.21.0"
```

Comment:
BTW, i published an updated version of our fork a while ago https://github.com/linkerd/webpki/tree/cert-dns-names-0.21